.github/ISSUE_TEMPLATE/reduce-code-complexity.md

```
@@ -0,0 +1,59 @@
+
+title: "[Refactor] Reduce Complexity in `<file>`"
+labels: [refactor, technical-debt, complexity]
+assignees: 
+  - ShafayetSadi
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **This issue is for reducing complexity in a file where SonarCloud or other analysis tools detected high cyclomatic complexity.**
+
+  - type: input
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)